### DISCUSSION TITLE
Automatic update

### DISCUSSION THREAD

**Discussion by Ananas0631** [fork-maintainers/iceraven-browser#807] [Ideas]
It would be great if there would be an update alert even if doesn't update by itself. There might be an extension for this either.

**Comment by ChiefMikeK** [comment]
FFUpdater |  https://f-droid.org/en/packages/de.marmaro.krt.ffupdater/

**Reply by Ananas0631** [reply]
Thanks a lot

**Reply by ydrk12** [reply]
Nice of you to find such a conveience

**Reply by HamzaaAkmal** [reply]
can anyone please help when i try to build apk using ./gradlew assembleDebug that there a miniconda task comes always and crashes the build i'm so coked please if someone knows how to fix that issue guide me it would be a great help! also if someone had glean free fork because won't need it

**Comment by Ekd-learning** [comment]
Obtanium (automatic GitHub apps updater): https://f-droid.org/packages/dev.imranr.obtainium.fdroid
